chevah_build

```diff
@@ -39,6 +40,13 @@ EXTRA_LIBRARIES="\
    python-modules/python-setproctitle-1.1.8.dev0 \
    "
 
+# We only use pip on ArchLinux and Windows builds
```

Comment:
it should not matter at this level how we use them... soon we will use them on RHEL.. and then we will forget to update this comment and an invalid commend is worst than no comment at all.

also, a comment is a sign of 'I have no idea how to better write this code so that it will speak for itself... so here is this comment which is valid for the time when I wrote this code, but I don't guarantee that it will be valid later'